src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/ClientCredentialTypeTests.cs

```diff
@@ -111,13 +157,35 @@ public static void BasicAuthenticationEmptyUser_throw_ArgumentException()
         Assert.True(exception.Message.ToLower().Contains(paraMessage), string.Format("Expected exception message to contain: '{0}', actual: '{1}'", paraMessage, exception.Message));
     }
 
-    [ConditionalFact(nameof(Domain_Joined),
-                     nameof(Root_Certificate_Installed),
-                     nameof(Digest_Authentication_Available), 
-                     nameof(Explicit_Credentials_Available))]
+#if FULLXUNIT_NOTSUPPORTED
+    [Fact]
+#else
+    [ConditionalFact(nameof(Server_Domain_Joined), nameof(Root_Certificate_Installed), nameof(Digest_Authentication_Available), nameof(Explicit_Credentials_Available))]
+#endif
```

Comment:
Non-blocking nit: I prefer the original indented style because the single line is too long to read.